src/app/dashboard/whitelist/actions.js

```diff
@@ -0,0 +1,281 @@
+//@flow
+
+import * as ui from 'polymath-ui'
+import { TransferManager, SecurityToken } from 'polymathjs'
+import type { Investor } from 'polymathjs/types'
+
+import { formName as userFormName } from './components/userForm'
+import { formName as editInvestorsFormName } from './components/editInvestorsForm'
+import type { GetState } from '../../../redux/reducer'
+import type { ExtractReturn } from '../../../redux/helpers'
+
+//ac_ = actionCreator
+export const TRANSFER_MANAGER = 'dashboard/whitelist/TRANSFER_MANAGER'
+export const ac_transferManager = (transferManager: TransferManager) => ({ type: TRANSFER_MANAGER, transferManager })
+
+export const UPLOAD_CSV = 'dashboard/whitelist/UPLOAD_CSV'
+export const ac_csvUpload = (csvMessage: string, addresses: Array<string>, sell: Array<string>, buy: Array<string>, previewCSVShowing: boolean, ) =>
+  ({ type: UPLOAD_CSV, csvMessage, addresses, sell, buy, previewCSVShowing })
+export const UPLOAD_CSV_FAILED = 'dashboard/whitelist/UPLOAD_CSV_FAILED'
+
+export const GET_WHITELIST = 'dashboard/whitelist/GET_WHITELIST'
+export const ac_getWhitelist = (investors: Array<Investor>) => ({ type: GET_WHITELIST, investors })
+// export const GET_WHITELIST_FAILED = 'dashboard/whitelist/GET_WHITELIST_FAILED'
+
+export const LIST_LENGTH = 'dashboard/whitelist/LIST_LENGTH'
+export const ac_listLength = (listLength: number) => ({ type: LIST_LENGTH, listLength })
```

Comment:
if you dispatch this action creators with ac_ prefix only once, then you don't need separate function for them.
just do `dispatch({ type: LIST_LENGTH, listLength })` in the listLength function for example